src/desktop/apps/article/templates/classic_meta.jade

```diff
@@ -1,6 +1,6 @@
 title= article.get('search_title') ? article.get('search_title') : article.get('thumbnail_title')
 link( rel="canonical" href=article.fullHref() )
-if article.hasAMP()
+if article.hasAMP() && (article.layout == "standard" || article.layout == "feature" || article.layout == "news")
```

Comment:
Is there a reason not to add this logic directly to `hasAMP()`?

https://github.com/artsy/force/blob/2bc3466f25328fb2a9956cb01a71e1e1f0799431/src/desktop/models/article.coffee#L222-L228

Reply:
`hasAMP`  is already checking for article layout being published, featured, and of layout type news || standard || feature, yet it still returns true for `hasAMP` for Vanguard articles, but it isn't clear to me why.  The logic here seems to be correct though, which is why I didn't modify this part.

Reply:
The data for Vanguard articles is `featured: true`, `layout: "series"`, `published: true`.

Reply:
Articles in the `classic` layout are only from internal/partner channels, so should already be excluded from AMP-- I don't think we will ever reach this template, the `hasAMP` may be legacy from before we moved articles to reaction.